docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -411,13 +441,19 @@ PublishModeQosPolicy
 This QoS Policy configures how the |DataWriter| sends the data.
 See |PublishModeQosPolicy-api|.
 
+It also configures the name of the flow controller to use when asynchronous publishing is used.
+It should be the name of a flow controller registered on the creation of the DomainParticipant.
+See |FlowControllersQos|.
+
 List of QoS Policy data members:
 
-+------------------------------------------+---------------------------------+-----------------------------------------+
-| Data Member Name                         | Type                            | Default Value                           |
-+==========================================+=================================+=========================================+
-| |PublishModeQosPolicy::kind-api|         | :ref:`publishmodeqospolicykind` | |SYNCHRONOUS_PUBLISH_MODE-api|          |
-+------------------------------------------+---------------------------------+-----------------------------------------+
++--------------------------------------------+---------------------------------+-----------------------------------------+
+| Data Member Name                           | Type                            | Default Value                           |
++============================================+=================================+=========================================+
+| |PublishModeQosPolicy::kind-api|           | :ref:`publishmodeqospolicykind` | |SYNCHRONOUS_PUBLISH_MODE-api|          |
```

Comment:
Following this link to the API Reference there is the following text:

```
Since

Functionality not implemented yet. Coming soon.
```

We should remove this doxygen comments from Fast DDS.